src/modules/fancyzones/editor/FancyZonesEditor/GridEditor.xaml.cs

```diff
@@ -137,80 +137,48 @@ private void ExtendRangeToHaveEvenCellEdges()
             // extend each edge of the [(_startCol, _startRow) - (_endCol, _endRow)] range based on merged cells until you have 4 straight edges with no "straddling cells"
```

Comment:
// extend each edge of the [(_startCol, _startRow) - (_endCol, _endRow)] range based on merged cells until you have 4 straight edges with no "straddling cells"

Duplicated comment with 3 rows below